gradle.properties

```diff
@@ -13,7 +13,7 @@ platformType=IC
 platformVersion=2023.2
 # Plugin Dependencies -> https://plugins.jetbrains.com/docs/intellij/plugin-dependencies.html
 # Example: platformPlugins = com.intellij.java, com.jetbrains.php:203.4449.22
-platformPlugins=com.redhat.devtools.intellij.telemetry:1.2.1.62, textmate, properties
+platformPlugins=com.redhat.devtools.intellij.telemetry:1.2.1.62, textmate, properties, com.intellij.modules.json
```

Comment:
Had to add a dependency on JSON, but it should be available in all JetBrains IDEs as I mentioned to you directly. I have the same type of dependency in my own commercial plugin, and I've already been reassured by JetBrains that it's fine in spite of their recent move to a distinct JSON plugin.